Meta/Lagom/Tools/CodeGenerators/LibUnicode/GenerateUnicodeData.cpp

```diff
@@ -760,9 +763,46 @@ static constexpr Array<Span<CodePointRange const>, @size@> @name@ { {)~~~");
     append_prop_list("s_scripts"sv, "s_script_{}"sv, unicode_data.script_list);
     append_prop_list("s_script_extensions"sv, "s_script_extension_{}"sv, unicode_data.script_extensions);
 
+    generator.append(R"~~~(
+struct CodePointName {
+    u32 code_point { 0 };
+    StringView name;
+    StringView u1_name;
+};
+)~~~");
+
+    generator.set("code_point_names_size", String::number(unicode_data.code_point_data.size()));
+    generator.append(R"~~~(
+static constexpr Array<CodePointName, @code_point_names_size@> s_code_point_names { {
+)~~~");
+    for (auto const& code_point_data : unicode_data.code_point_data) {
+        generator.set("code_point", String::formatted("{:#x}", code_point_data.code_point));
+        generator.set("code_point_name", code_point_data.name);
+        // A lot of the time, the "Unicode 1 name" is empty. Save space by using null StringViews.
```

Comment:
:thinking: what's the alternative here? 
if it's so sparse, perhaps it could use a separate array with sparse entries instead? the search would be the same binary search, so likely no need for further change.

Reply:
Since we now have only `display_name`, this has become irrelevant.